packages/fxa-payments-server/server/lib/server.js

```diff
@@ -22,6 +23,21 @@ module.exports = () => {
 
   const app = express();
 
+  // Each of these config values (e.g., 'servers.content') will be exposed as the given
+  // variable to the client/browser (via fxa-content-server/config)
+  const CLIENT_CONFIG_MAP = {
+    contentUrl: 'servers.content',
+    oAuthUrl: 'servers.oauth',
+    profileUrl: 'servers.profile',
+  };
+
+  // This is a list of all the paths that should resolve to index.html:
+  const INDEX_ROUTES = [
+    '/',
+    '/subscriptions',
+    '/products/:productId',
+  ];
```

Comment:
This needs to stay in sync with React Routes, but I didn't do anything to accomplish that beyond comments

Reply:
There might be a way to do this programmatically, but heck if I know how off the top of my head

Reply:
I guess a shared file they both load that lists the routes, but I think that cure is worse than the disease